redaxo/src/addons/debug/install.php

```diff
@@ -1,13 +1,36 @@
 <?php
 
 $addon = rex_addon::get('debug');
-rex_dir::copy(
-    $addon->getPath('frontend/web/'),
-    $addon->getAssetsPath('clockwork')
-);
 
-$indexPath = $addon->getAssetsPath('clockwork/index.html');
+// the filenames contain rev hashes and the old ones would never be cleaned up
+rex_dir::delete($addon->getAssetsPath());
 
-$index = file_get_contents($indexPath);
-$index = preg_replace('/(href|src)=("?)([^>\s]+)/', '$1=$2'.$addon->getAssetsUrl('clockwork/$3'), $index);
-file_put_contents($indexPath, $index);
+// extract clockwork frontend
+$zipArchive = new ZipArchive();
+
+// use path relative to __DIR__ to get correct path in update temp dir
+$path = __DIR__.'/frontend/frontend.zip';
+
+$message = '';
+try {
+    if (true === $zipArchive->open($path) &&
+        true === $zipArchive->extractTo($addon->getAssetsPath('clockwork'))
+    ) {
+        $zipArchive->close();
+
+        $indexPath = $addon->getAssetsPath('clockwork/index.html');
+
+        $index = file_get_contents($indexPath);
+        $index = preg_replace('/(href|src)=("?)([^>\s]+)/', '$1=$2'.$addon->getAssetsUrl('clockwork/$3'), $index);
+        file_put_contents($indexPath, $index);
+    } else {
+        $message = rex_i18n::msg('debug_error_unzip') . '<br>' . $path;
+    }
+} catch (Exception $e) {
+    $message = rex_i18n::msg('debug_error_unzip') . '<br>' . $path;
+    $message .= '<br>' . $e->getMessage();
```

Comment:
ich wundere mich grad dass wir hier html in die exception message schreiben.

aber es ist an anderen stellen auch so, daher für diesen PR nicht relevant.